apiserver/facades/controller/firewaller/register.go

```diff
@@ -58,6 +58,8 @@ func newFirewallerAPIV7(ctx facade.ModelContext) (*FirewallerAPI, error) {
 		domainServices.Config(),
 		domainServices.Application(),
 		domainServices.Machine(),
+		domainServices.Controller(),
+		ctx.ModelUUID(),
```

Comment:
I believer this will go away as well if we wire up the model info service for the is controller question.

Reply:
Sure changed this to model info service, thanks tom!